### PR TITLE
[SMALLFIX] Remove unused parameter when constructing LocalTachyonClusterResource

### PR DESCRIPTION
**Pull request by apc999**

### CODE REVIEW

File: tests/src/test/java/tachyon/LocalTachyonClusterResource.java [hunk 1]
```diff
@@ -199,7 +195,8 @@ public void evaluate() throws Throwable {
    */
   @Retention(RetentionPolicy.RUNTIME)
   public @interface Config {
-    String[] tachyonConfParams() default {};
+    String[]tachyonConfParams() default {};
```
Comment (jsimsa) [inline tests/src/test/java/tachyon/LocalTachyonClusterResource.java]: missing space?

**Comment by jsimsa** [comment]
one minor comment

**Comment by apc999** [comment]
@jsimsa PTAL, thanks

**Comment by apc999** [comment]
Jenkins, test this please

**Comment by aaudiber** [comment]
LGTM

**Pull request merged**